CHANGELOG.md

```diff
@@ -447,6 +450,12 @@ and CVE-2018-14567)
 - Fix a locale select bug to make the options visible on a dark navbar.
 - Some improvements to fix some issues reported by codeclimate.
 
+<<<<<<< HEAD
```

Comment:
:eyes: I think removing this conflict tag is a good idea :thinking:

Reply:
Oh, yes, this is a brilliant idea. :stuck_out_tongue: